modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/qpid/ServerConfiguration.java

```diff
@@ -264,19 +270,29 @@ private static void substituteEnvironmentVariables(org.apache.commons.configurat
         }
     }
 
-    private static org.apache.commons.configuration.Configuration parseConfig(File file) throws ConfigurationException {
-        ConfigurationFactory factory = new ConfigurationFactory();
-        factory.setConfigurationFileName(file.getAbsolutePath());
-        org.apache.commons.configuration.Configuration conf = factory.getConfiguration();
+    private static org.apache.commons.configuration2.Configuration parseConfig(File file) throws ConfigurationException {
```

Comment:
Shall we add a javadoc here?

Reply:
Added with https://github.com/wso2/andes/pull/1070/commits/9d54bd823e23ecf281fa6f8fc624899f5e5a8b24